tests/acceptance/bootstrap/FilesVersionsContext.php

```diff
@@ -547,19 +567,17 @@ public function listVersionFolder(
 	}
 
 	/**
-	 * This will run before EVERY scenario.
-	 * It will set the properties for this object.
+	 * @When user :user gets the number of versions of file :file from space :space
+	 * @When user :user tries to get the number of versions of file :file from space :space
 	 *
-	 * @BeforeScenario
-	 *
-	 * @param BeforeScenarioScope $scope
+	 * @param string $user
+	 * @param string $file
+	 * @param string $space
 	 *
 	 * @return void
 	 */
-	public function before(BeforeScenarioScope $scope):void {
-		// Get the environment
-		$environment = $scope->getEnvironment();
-		// Get all the contexts you need in this context
-		$this->featureContext = BehatHelper::getContext($scope, $environment, 'FeatureContext');
+	public function userGetsTheNumberOfVersionsOfFileFromSpace(string $user, string $file, string $space): void {
+		$fileId = $this->spacesContext->getFileId($user, $space, $file);
+		$this->userGetsTheNumberOfVersionsOfFileOfTheSpace($user, $fileId);
```

Comment:
I would not recommend to use one step def inside another. 
```suggestion
		$this->featureContext->setResponse(
			$this->featureContext->makeDavRequest(
				$user,
				"PROPFIND",
				$fileId,
				null,
				null,
				null,
				"versions"
			)
		);
```